app-text/dblatex/dblatex-0.3.12.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="DocBook to LaTeX/ConTeXt Publishing"
+DESCRIPTION="dblatex is a program that transforms your SGML/XML DocBook \
+documents to DVI, PostScript or PDF by translating them into pure LaTeX as a \
+first process. MathML 2.0 markups are supported, too. It started as a clone \
+of DB2LaTeX."
+HOMEPAGE="http://dblatex.sourceforge.net/"
+COPYRIGHT="2003–2020 Benoit Guillon"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/project/dblatex/dblatex/dblatex-$portVersion/dblatex3-$portVersion.tar.bz2"
+CHECKSUM_SHA256="16e82786272ed1806a079d37914d7ba7a594db792dc4cc34c1c3737dbd4da079"
+SOURCE_DIR="dblatex3-$portVersion"
+
+ARCHITECTURES="?any"
+
+PROVIDES="
+	dblatex = $portVersion
+	cmd:dblatex
+	"
+REQUIRES="
+	haiku
+	cmd:convert
+	cmd:epstopdf
+	cmd:kpsewhich
+	cmd:latex
+	cmd:makeindex
+	cmd:pdflatex
+	cmd:python3
+	texlive_texmf # TODO: this must be changed when the texlive_texmf package is split up
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	cmd:convert
+	cmd:epstopdf
+	cmd:kpsewhich
+	cmd:latex
+	cmd:makeindex
+	cmd:pdflatex	
+	setuptools_python3
+	texlive_texmf
```

Comment:
texlive_texmf is gcc8 so architecture can't be "any"
```diff
- 	texlive_texmf
+ 	texlive${secondaryArchSuffix}_texmf
```

Reply:
texlive_texmf technically is "any", but its name is different on x86. Maybe we should add an additional PROVIDES to texlive${secondaryArchSuffix}_texmf without the "secondaryArchSuffix" part if it is not empty?

To be honest, this could even be viewed as a "bug" in haikuporter. Why does the package name of the subpackage have to contain "_x86" if it is architecture "any"?

Or could something like `SECONDARY_ACHITECTURES_texmf=""` be set in the texlive recipe (explicitly clearing the secondary architectures for the subpackages)?

Reply:
Need to search if there is some example here on haikuports that uses such an extention, at this moment I don't know if it's possible

Reply:
Maybe libreoffice? I didn't look specifically at it, but I think it contains "any" subpackages.

Reply:
Provides there is also secondary:
https://github.com/haikuports/haikuports/blob/14c2cab5428145b93232cb69683a67bbe68a9f06/app-office/libreoffice/libreoffice-6.4.7.2.recipe#L146

Reply:
So far I haven't found anything that could work here, I did something similar for the fpc package, but that included a separate package, so it would mean that in this case you should split up texlive and texlive_texmf into seperate packages ... one benefit maybe, the base package including the commands for latex could be included in haikuports as active (I guess there are more packages around the size of +180MiB already :)

Reply:
I will try splitting the texlive recipe in two, that is probably a good idea anyway. But please be patient, this will take a while ;)